### PR TITLE
Update ch.pronatura.waldreservate

### PR DESCRIPTION
**Pull request by faselm**
Test link: https://mf-chsdi3.dev.bgdi.ch/ltfas_pronatura_waldreservate/shorten/8ff6c3dc61

### CODE REVIEW

File: chsdi/models/vector/dritte.py [hunk 1]
```diff
@@ -72,6 +72,11 @@ class PronaturaWaldreservate(Base, Vector):
     gesflaeche = Column('obj_gesflaeche', Float)
     gisteilobjekt = Column('obj_gisteilobjekt', Float)
     mcpfe = Column('mcpfe', Unicode)
+    mcpfe_class_de = Column('mcpfe_class_de', Unicode)
+    mcpfe_class_fr = Column('mcpfe_class_fr', Unicode)
+    mcpfe_class_it = Column('mcpfe_class_it', Unicode)
+    mcpfe_class_en = Column('mcpfe_class_en', Unicode)
+    mcpfe_class_rm = Column('mcpfe_class_rm', Unicode)
```
Comment (b4l) [inline chsdi/models/vector/dritte.py:79]: Why specify rm but in the mako you map rm to de?

**Review by ltclm — CHANGES_REQUESTED** [review]
thanks for this @faselm.
same here, either remove rm from model and table or fill it with data and remove the mapping.

File: chsdi/templates/htmlpopup/pronatura.mako [hunk 2]
```diff
@@ -1,18 +1,18 @@
 <%inherit file="base.mako"/>
 
 <%def name="table_body(c, lang)">
-    <tr><td class="cell-left">${_('ch.pronatura.waldreservate.objnummer')}</td>                         <td>${c['attributes']['objnummer'] or '-'}</td></tr>
-    <tr><td class="cell-left">${_('ch.pronatura.waldreservate.name')}</td>                         <td>${c['attributes']['name'] or '-'}</td></tr>
```
Comment (rebert) [inline chsdi/templates/htmlpopup/pronatura.mako:5]: name is the only Declaration that is being translated to rm (NOme)... which is not consistent

**Review by rebert — CHANGES_REQUESTED** [review]
Nice. Only a translation the rm translation of ch.pronatura.waldreservate.name to adapt

**Comment by faselm** [comment]
The rm translation has been fixed. However, it seems there are many inconsistencies in the translations for this layer and the bafu one, even on prod. In German, there is "Teilfläche Perimeter [ha]". Unless I'm missing something in German, perimeter in hectars doesn't make sense and the item in translated as "Area" and "Surface in English and French. Also as "Fläche" in rm... Not sure how all this should be fixed. Maybe you have an idea or I will also ask Mario...

**Comment by ltclm** [comment]
@faselm you can fix the translations in the bod in the table ms_item:
```SQL
select * FROM ms_item where fk_layer_name like '%waldreservate' order by 3,2
```

the translations with for the attribute gisteilobjekt are empty there.

**Review by ltclm — CHANGES_REQUESTED** [review]
looks good to me.
the attribute mcpfe is obviously not used anymore in chsdi. 
If thats correct, please remove it in this pr from model/mako. Since it is still used in the wms mapfile we will have to keep it in the table.

**Comment by ltclm** [comment]
you would have to remove it from __queryable_attributes__ too, but please ask mario first.

**Comment by faselm** [comment]
It's OK for Mario to remove mcpfe

File: chsdi/models/vector/dritte.py [hunk 3]
```diff
@@ -63,15 +63,18 @@ class PronaturaWaldreservate(Base, Vector):
     __table_args__ = ({'schema': 'pronatura', 'autoload': False})
     __template__ = 'templates/htmlpopup/pronatura.mako'
     __bodId__ = 'ch.pronatura.waldreservate'
-    __queryable_attributes__ = ['name', 'objnummer', 'gesflaeche', 'gesflaeche', 'gisteilobjekt', 'mcpfe']
+    __queryable_attributes__ = ['name', 'objnummer', 'gesflaeche', 'gesflaeche', 'gisteilobjekt']
```
Comment (ltclm) [inline chsdi/models/vector/dritte.py:66]: you can remove the whole line from both layers ch.pronatura.waldreservate/ch.bafu.waldreservate, this feature has not been used at all in 2020 and 2021.

**Review by ltclm — CHANGES_REQUESTED** [review]

**Review by ltclm — APPROVED** [review]

**Pull request merged**